--- /home/chilicuil/.vim/plugin/debugger.py.orig	2009-08-31 01:41:07.000000000 -0500
+++ /home/chilicuil/.vim/plugin/debugger.py	2009-08-31 01:44:54.000000000 -0500
@@ -413,15 +413,15 @@
   def on_create(self):
     self.write(                                                          \
         '[ Function Keys ]                 |                       \n' + \
-        '  <F1>   resize                   | [ Normal Mode ]       \n' + \
-        '  <F2>   step into                |   ,e  eval            \n' + \
-        '  <F3>   step over                |                       \n' + \
-        '  <F4>   step out                 |                       \n' + \
-        '  <F5>   run                      | [ Command Mode ]      \n' + \
-        '  <F6>   quit debugging           | :Bp toggle breakpoint \n' + \
-        '                                  | :Up stack up          \n' + \
-        '  <F11>  get all context          | :Dn stack down        \n' + \
-        '  <F12>  get property at cursor   |                       \n' + \
+        ' <Leader><F1>   resize                   | [ Normal Mode ]       \n' + \
+        ' <Leader><F2>   step into                |   ,e  eval            \n' + \
+        ' <Leader><F3>   step over                |                       \n' + \
+        ' <Leader><F4>   step out                 |                       \n' + \
+        ' <Leader><F5>   run                      | [ Command Mode ]      \n' + \
+        ' <Leader><F6>   quit debugging           | :Bp toggle breakpoint \n' + \
+        '                                         | :Up stack up          \n' + \
+        ' <Leader><F11>  get all context          | :Dn stack down        \n' + \
+        ' <Leader><F12>  get property at cursor   |                       \n' + \
         '\n')
     self.command('1')
 
